docs/content/deprecation/releases.md

```diff
@@ -6,7 +6,8 @@ Below is a non-exhaustive list of versions and their maintenance status:
 
 | Version | Release Date | Active Support     | Security Support | 
 |---------|--------------|--------------------|------------------|
-| 2.10    | Apr 24, 2023 | Yes                | Yes              |
+| 2.11    | Feb 12, 2023 | Yes                | Yes              |
+| 2.10    | Apr 24, 2023 | Ended Feb 12, 2023 | No               |
```

Comment:
Late (already merged), but I think "Feb 12, **2024**" was meant here? :)

Reply:
copy-pasting mistake :smile: 
Fixed in #10449
Thank you :+1: